pkg/api/v1/types.go

```diff
@@ -295,6 +295,8 @@ type VirtualMachineInstanceNetworkInterface struct {
 	IP string `json:"ipAddress,omitempty"`
 	// Hardware address of a Virtual Machine interface
 	MAC string `json:"mac,omitempty"`
+	// Name of the interface, corresponds to the pod bridge the interface is
+	Name string `json:"name,omitempty"`
```

Comment:
The here is actually the name of the bridge to which the vnic is connected on the pod.
Should we name it BridgeName?

Reply:
Yeah, that's a misunderstanding. I meant the name of the volume/interface, and not the name of the bridge. I hope my review clarifies that. Let me know if you have further questions.